rpc/src/rpc_subscriptions.rs

```diff
@@ -2802,6 +2804,15 @@ pub(crate) mod tests {
                "subscription": 0,
            }
         });
+        *expected
+            .get_mut("params")
+            .unwrap()
+            .get_mut("result")
+            .unwrap()
+            .get_mut("value")
+            .unwrap()
+            .get_mut("rentEpoch")
+            .unwrap() = serde_json::to_value(u64::MAX).unwrap();
```

Comment:
why not
```
"rentEpoch": u64::MAX,
```
in json thing above?

Reply:
looks like json! macro tries to read it as 32-bit integer and it errors that it is too big. I could also escape the string and use a literal value and use the same `serde_json::from_str::<serde_json::Value>`. This seemed 'better'?

Reply:
That works. So does `...615_usize`
Doh. Rookie rust mistake. I'm so used to rust just picking the right types for me. I would not have guessed it would default to i32.
```
error: literal out of range for `i32`
    --> rpc/src/rpc_subscriptions.rs:2800:37
     |
2800 |                        "rentEpoch": 18_446_744_073_709_551_615,
     |                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^
     |
     = note: `#[deny(overflowing_literals)]` on by default
     = note: the literal `18_446_744_073_709_551_615` does not fit into the type `i32` whose range is `-2147483648..=2147483647`
     = help: consider using the type `i128` instead
     ```